extension/src/repository/commands/register.ts

```diff
@@ -37,14 +37,14 @@ const registerResourceGroupCommands = (
   repositories: WorkspaceRepositories,
   internalCommands: InternalCommands
 ) => {
-  internalCommands.registerExternalCommand<Root>(
-    RegisteredCommands.GIT_STAGE_ALL,
-    getStageAllCommand(repositories)
+  internalCommands.registerExternalCliCommand<Root>(
+    RegisteredCliCommands.GIT_STAGE_ALL,
+    getStageAllCommand(repositories, internalCommands)
```

Comment:
[F] The extension will now "offer to show errors" (open the output channel) if these commands fail.